sample/HelloWorld/public/js/helloworld.js

```diff
@@ -1,29 +1,22 @@
-// Initialize an OpenTok Session object
+// Initialize an OpenTok Session object.
 var session = OT.initSession(sessionId);
 
-// Initialize a Publisher, and place it into the element with id="publisher"
+// Initialize a Publisher, and place it into the 'publisher' DOM element.
 var publisher = OT.initPublisher(apiKey, 'publisher');
 
-// Attach an event handler for when the session dispatches the 'streamCreated' event.
 session.on('streamCreated', function(event) {
-    // This function runs when another client publishes a stream (eg. session.publish())
-
-    // Subscribe to the stream that caused this event, put it inside the DOM element with id='subscribers'
-    session.subscribe(event.stream, 'subscribers', {
-        insertMode: 'append'
-    });
+  // Called when another client publishes a stream.
+  // Subscribe to the stream that caused this event.
+  session.subscribe(event.stream, 'subscribers', { insertMode: 'append' });
 });
 
-// Connect to the Session using the 'apiKey' of the application and a 'token' for permission
+// Connect to the session using your OpenTok API key and the client's token for the session
```

Comment:
I usually try to capitalize Session because i think of it as a proper noun within OpenTok as opposed to the general meaning of the word "session".

"your OpenTok API key" is less clear, you might have many of them. there is one for this project, and in fact if we're going to revise the terminology, we should move to Project since that's what is in Account Portal.

the word "permission" was important because it helps build understanding about what a token is functionally worth.